easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.3-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,46 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# Updated: Pavel Grochal (INUITS)
+# License: GPLv2
+
+# "make install" doesnt copy all the compiled binaries so we use the "MakeCp" easyblock
+# to be sure everything is copied and we run ./configure in prebuildopts
+easyblock = 'MakeCp'
+
+name = 'IDBA-UD'
+version = '1.1.3'
+
+homepage = 'https://i.cs.hku.hk/~alse/hkubrg/projects/idba_ud/'
+description = """ IDBA-UD is a iterative De Bruijn Graph De Novo Assembler for Short Reads
+ Sequencing data with Highly Uneven Sequencing Depth. It is an extension of IDBA algorithm.
+ IDBA-UD also iterates from small k to a large k. In each iteration, short and low-depth
+ contigs are removed iteratively with cutoff threshold from low to high to reduce the errors
+ in low-depth and high-depth regions. Paired-end reads are aligned to contigs and assembled
+ locally to generate some missing k-mers in low-depth regions. With these technologies, IDBA-UD
+ can iterate k value of de Bruijn graph to a very large value with less gaps and less branches
+ to form long contigs in both low-depth and high-depth regions."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
```

Comment:
@Darkless012 From the description, this seems like it's performance-sensitive, so with that in mind `GCC` is a better choice for the toolchain (since that's at the same level as `iccifort` when compiling with Intel compilers)

Reply:
Changed to GCC, but from what I can tell, running tests on GCC vs GCCcore didn't have any difference in time.

Reply:
this could probably use some clarification - using `GCC` instead of `GCCcore` for "performance-sensitive" software is not because there is any difference in performance between `GCC` and `GCCcore` (there isn't), it's to make it easy to use a different compiler toolchain. I suppose we should say "for performance-sensitive software for which we expect using a compiler toolchain other than `GCC` would have a significant performance difference" (?)